crates/router/src/core/payments/flows.rs

```diff
@@ -762,6 +771,7 @@ macro_rules! default_imp_for_payouts {
 impl<const T: u8> api::Payouts for connector::DummyConnector<T> {}
 
 default_imp_for_payouts!(
+    connector::Square,
```

Comment:
Add in Alphabetical order, please check for all the default `impl` in this file

Reply:
Done